ComponentKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ComponentKit'
-  s.version = '0.22'
+  s.version = '0.22.1'
```

Comment:
The new version will contain some breaking changes in the framework, so I would rather make it 0.23 instead

Reply:
I would think Yoga 1.6.0 to 1.x shouldn’t have any breaking API changes. At the end, the third digit never involves api changes, it just for bug fixes. If that is not true in yoga’s case, then I can change the version to 0.23

Ideally, breaking API changes should only happen on major versions. In this scenario, if our dependency was moving from yoga 1.6 to 2.0

Reply:
Yoga doesn't, but the ComponentKit does. So it's better to be safe